### ISSUE TITLE
Command errored out with exit status 1: while installing the package in python 3.9.7 

### ISSUE THREAD

**Issue by ndganesan** [gboeing/osmnx#772]
**Problem description**
  I'm trying to install the library, installation error, it trying to install geopandas package it requires Fiona package that is not installing  

Command errored out with exit status 1
'C:\Users\Ganesan\AppData\Local\Programs\Python\Python39\python.exe' -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Ganesan\\AppData\\Local\\Temp\\pip-install-vrsk2ptc\\fiona_1904346d80c14cd590ab768854e28479\\setup.py'"'"'; __file__='"'"'C:\\Users\\Ganesan\\AppData\\Local\\Temp\\pip-install-vrsk2ptc\\fiona_1904346d80c14cd590ab768854e28479\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\Ganesan\AppData\Local\Temp\pip-record-z3sa4xvp\install-record.txt' --single-version-externally-managed --compile --install-headers 'C:\Users\Ganesan\AppData\Local\Programs\Python\Python39\Include\fiona' Check the logs for full command output.

**Environment information**
   Windows 10
  Python 3.9.7
<details>
Installing collected packages: Shapely, pyproj, fiona, Rtree, geopandas, descartes, osmnx
analytics-python                  1.4.0
anyio                             3.3.3
argon2-cffi                       21.1.0
arrow                             1.2.0
atomicwrites                      1.4.0
attrs                             20.3.0
Automat                           20.2.0
Babel                             2.9.1
backcall                          0.2.0
backoff                           1.10.0
backports.entry-points-selectable 1.1.0
bcrypt                            3.2.0
beautifulsoup4                    4.10.0
bleach                            4.1.0
bob                               9.0.0
bokeh                             2.4.0
boto3                             1.19.8
botocore                          1.22.8
Brotli                            1.0.9
CacheControl                      0.12.9
cachetools                        4.2.4
cachy                             0.3.0
certifi                           2021.10.8
cffi                              1.14.6
chainer                           7.8.0
chardet                           4.0.0
charset-normalizer                2.0.7
cirq                              0.12.0
cirq-aqt                          0.12.0
cirq-core                         0.12.0
cirq-google                       0.12.0
cirq-ionq                         0.12.0
cirq-pasqal                       0.12.0
cirq-rigetti                      0.12.0
cirq-web                          0.12.0
clang                             11.0
cleo                              0.8.1
click                             8.0.3
click-plugins                     1.1.1
cligj                             0.7.2
clikit                            0.6.2
cloudpickle                       2.0.0
colorama                          0.4.4
constantly                        15.1.0
crashtest                         0.3.1
cryptography                      35.0.0
cssselect                         1.1.0
cycler                            0.10.0
Cython                            0.29.24
dash                              2.0.0
dash-core-components              2.0.0
dash-html-components              2.0.0
dash-table                        5.0.0
debugpy                           1.5.0
decorator                         4.4.2
defusedxml                        0.7.1
Delorean                          1.0.0
dill                              0.3.4
distlib                           0.3.3
dm-sonnet                         2.0.0
dm-tree                           0.1.6
easydict                          1.9
ebbe                              1.6.0
eli5                              0.11.0
emoji                             1.6.0
entrypoints                       0.3
fabric                            2.6.0
ffmpy                             0.3.0
filelock                          3.3.0
fire                              0.4.0
flashtext                         2.7
Flask                             2.0.2
Flask-CacheBuster                 1.0.0
Flask-Compress                    1.10.1
Flask-Cors                        3.0.10
Flask-Login                       0.5.0
flatbuffers                       2.0
fog                               0.11.7
future                            0.18.2
gast                              0.5.2
gensim                            4.1.2
glog                              0.3.1
google-api-core                   2.1.0
google-auth                       2.3.0
googleapis-common-protos          1.53.0
gradio                            2.3.7
graphviz                          0.17
greenlet                          1.1.2
h11                               0.9.0
h2                                3.2.0
h3                                3.7.3
h5py                              3.4.0
hpack                             4.0.0
html5lib                          1.1
httpcore                          0.11.1
httpx                             0.15.5
humanize                          3.12.0
hyperframe                        6.0.1
hyperlink                         21.0.0
hypothesis                        6.23.2
idna                              3.3
imageio                           2.9.0
imageio-ffmpeg                    0.4.5
importlib-metadata                4.8.1
incremental                       21.3.0
iniconfig                         1.1.1
invoke                            1.6.0
ipykernel                         6.4.1
ipython                           7.28.0
ipython-genutils                  0.2.0
ipywidgets                        7.6.5
iso8601                           0.1.16
itemadapter                       0.4.0
itemloaders                       1.0.4
itsdangerous                      2.0.1
jedi                              0.18.0
Jinja2                            3.0.2
jmespath                          0.10.0
joblib                            1.1.0
jsonschema                        4.1.0
jupyter                           1.0.0
jupyter-client                    7.0.6
jupyter-console                   6.4.0
jupyter-core                      4.8.1
jupyterlab-pygments               0.1.2
jupyterlab-widgets                1.0.2
keras                             2.7.0
keyring                           23.2.1
kiwisolver                        1.3.2
lark                              0.11.3
lightgbm                          3.3.0
linecache2                        1.0.0
lockfile                          0.12.2
luminoth                          0.2.3
lxml                              4.6.3
mahotas                           1.4.11
markdown2                         2.4.1
MarkupSafe                        2.0.1
matplotlib                        3.4.3
matplotlib-inline                 0.1.3
mistune                           0.8.4
monotonic                         1.6
moviepy                           1.0.3
mpmath                            1.2.1
msgpack                           1.0.2
munch                             2.5.0
nbclient                          0.5.4
nbconvert                         6.2.0
nbformat                          5.1.3
nest-asyncio                      1.5.1
networkx                          2.6.3
nltk                              3.6.5
nose                              1.3.7
notebook                          6.4.4
numpy                             1.21.3
oauthlib                          3.1.1
opencv-python                     4.5.3.56
packaging                         21.0
pandas                            1.3.3
pandocfilters                     1.5.0
paramiko                          2.8.0
parsel                            1.6.0
parso                             0.8.2
pastel                            0.2.1
pathlib2                          2.3.6
patsy                             0.5.2
pendulum                          2.1.2
pexpect                           4.8.0
phylactery                        0.2.3
pickleshare                       0.7.5
Pillow                            8.3.2
pip                               21.3.1
pip-review                        1.1.0
pipenv                            2021.5.29
pkginfo                           1.7.1
platformdirs                      2.4.0
plotly                            5.3.1
pluggy                            1.0.0
poetry                            1.1.11
poetry-core                       1.0.7
priority                          2.0.0
proglog                           0.1.9
prometheus-client                 0.11.0
prompt-toolkit                    3.0.20
Protego                           0.1.16
protobuf                          3.18.1
ptyprocess                        0.7.0
py                                1.10.0
pyasn1                            0.4.8
pyasn1-modules                    0.2.8
PyBrain                           0.3
pycparser                         2.20
pycryptodome                      3.11.0
pydantic                          1.8.2
PyDispatcher                      2.0.5
pydub                             0.25.1
pygame                            2.0.2
pyglet                            1.5.21
Pygments                          2.10.0
PyJWT                             1.7.1
pylev                             1.4.0
PyNaCl                            1.4.0
pyOpenSSL                         21.0.0
pyparsing                         2.4.7
pyproj                            3.2.1
PyQt5                             5.15.4
PyQt5-Qt5                         5.15.2
PyQt5-sip                         12.9.0
pyquil                            3.0.1
pyrsistent                        0.18.0
pytest                            6.2.5
python-dateutil                   2.8.2
python-gflags                     3.1.2
python-nvd3                       0.15.0
python-rapidjson                  1.4
python-slugify                    5.0.2
pytil                             7.0.0
pytz                              2021.3
pytzdata                          2020.1
pywin32                           302
pywin32-ctypes                    0.2.0
pywinpty                          1.1.4
PyYAML                            5.4.1
pyzmq                             22.3.0
qcs-api-client                    0.12.0
qtconsole                         5.1.1
QtPy                              1.11.2
queuelib                          1.6.2
ramp-workflow                     0.4.0
regex                             2021.10.8
requests                          2.26.0
requests-oauthlib                 1.3.0
requests-toolbelt                 0.9.1
retry                             0.9.2
retrying                          1.3.3
rfc3339                           6.2
rfc3986                           1.5.0
rpcq                              3.9.2
rsa                               4.7.2
ruamel.yaml                       0.17.16
ruamel.yaml.clib                  0.2.6
s3transfer                        0.5.0
scikit-learn                      1.0
scikit-video                      1.1.11
scipy                             1.7.1
Scrapy                            2.5.1
seaborn                           0.11.2
Send2Trash                        1.8.0
service-identity                  21.1.0
setuptools                        58.5.2
Shapely                           1.8.0
shellingham                       1.4.0
simpy                             4.0.1
six                               1.15.0
smart-open                        5.2.1
sniffio                           1.2.0
sortedcontainers                  2.4.0
soupsieve                         2.2.1
SQLAlchemy                        1.4.25
statsmodels                       0.13.0
sympy                             1.9
tabulate                          0.8.9
tenacity                          8.0.1
tensorboard-data-server           0.6.1
tensorboard-plugin-wit            1.8.0
terminado                         0.12.1
testpath                          0.5.0
text-unidecode                    1.3
textblob                          0.15.3
Theano                            1.0.5
threadpoolctl                     3.0.0
toml                              0.10.2
tomlkit                           0.7.2
torch                             1.9.1+cu111
torchaudio                        0.9.1
torchvision                       0.10.1+cu111
tornado                           6.1
tqdm                              4.62.3
traceback2                        1.4.0
traitlets                         5.1.0
Twisted                           21.7.0
twisted-iocpsupport               1.0.2
typing-extensions                 3.10.0.2
tzdata                            2021.2.post0
tzlocal                           3.0
Unidecode                         1.3.2
unittest2                         1.1.0
urllib3                           1.26.7
virtualenv                        20.8.1
virtualenv-clone                  0.5.7
w3lib                             1.22.0
wcwidth                           0.2.5
webencodings                      0.5.1
Werkzeug                          2.0.2
wheel                             0.37.0
widgetsnbextension                3.5.1
wxPython                          4.1.1
yafs                              0.3.0
zipp                              3.6.0
zope.interface                    5.4.0
</details>

**Provide a complete minimal reproducible example**
help me to install this package because it helps me run my project
```python
pip install -r requirements.txt
pip install geopandas

```

**Comment by gboeing** [comment]
Please reread the installation instructions https://osmnx.readthedocs.io/en/stable/#installation
